arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/list/monad/ListKMonad.kt

```diff
@@ -221,7 +221,7 @@ fun <A, B> List<A>.mproduct(arg1: Function1<A, Kind<ForListK, B>>): List<Tuple2<
   "EXTENSION_SHADOWED_BY_MEMBER",
   "UNUSED_PARAMETER"
 )
-@Deprecated("@extension projected functions are deprecated", ReplaceWith("ifM(arg1, arg2)", "arrow.core.ifM"))
+@Deprecated("@extension projected functions are deprecated", ReplaceWith("flatMap { bool -> if (bool) arg1() else arg2() }"))
```

Comment:
I suppose in this case no need for `flatMap` imports if it gets it from the std lib Iterable extensions.

Reply:
Exactly `flatMap` from Kotlin Std doesn't require an import.